.github/workflows/pr.yml

```diff
@@ -28,11 +28,11 @@ jobs:
       - name: Install Cargo toolchain
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: nightly
+          toolchain: nightly-2025-04-01
```

Comment:
toolchain chainges should have been merged in the `[solana-march-audit-2](https://github.com/across-protocol/contracts/tree/solana-march-audit-2)` branch already